### DISCUSSION TITLE
How to enable font aliasing?

### DISCUSSION THREAD

**Discussion by augustocdias** [wezterm/wezterm#2169] [Q&A]
The fonts look very ugly in comparison to Alacritty and iTerm. I have played with the `freetype_load_target` and the `freetype_render_target` options with no success

![image](https://user-images.githubusercontent.com/1080789/175552202-df1eedc1-16de-46c4-9a4d-4fc54059785c.png)

**Comment by wez** [comment]
Possibly the same answer as https://github.com/wez/wezterm/discussions/2231#discussioncomment-3104855 ?

**Reply by augustocdias** [reply]
I'm not seeing any difference... Are those options really in the root of the configuration? 

It does look okay on a Retina display, but it is not my main display usually :/

**Reply by wez** [reply]
It's a bit awkward to look into this; the screen shot is predominantly not wezterm and it's really hard to see without opening it full screen. You haven't called out explicitly which parts you find ugly. It looks like it might be the italics?  There's no config text to copy and paste and I don't know what version you are running.  Could you make a pass over this and fill in more details?  Using https://github.com/wez/wezterm/issues/new?assignees=&labels=bug&template=bug.yml can guide you in providing more actionable information and make it easier to follow up!

I would also suggest that you run through https://wezfurlong.org/wezterm/config/fonts.html#troubleshooting-fonts to get some more info.  There's a section at the bottom that can explain the shaping plan for some input text. I would suggest setting your main `font` setting to match the list of fonts you've set for italics if it is the italics that need troubleshooting.
